### PR TITLE
Let it be a bug only when it's reproducible with official runner image

### PR DESCRIPTION
**Pull request by mumoshu**
A custom runner image can break runners and ARC in interesting ways. Probably it's better to clearly state that ARC is not guaranteed to work with every custom runner image in the wild.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -7,10 +7,12 @@ body:
   id: read-troubleshooting-guide
   attributes:
     label: Checks
-    description: Please check the boxes below before submitting
+    description: Please check all the boxes below before submitting
     options:
     - label: I've already read https://github.com/actions-runner-controller/actions-runner-controller/blob/master/TROUBLESHOOTING.md and I'm sure my issue is not covered in the troubleshooting guide.
       required: true
+    - label: I'm not using a custom runner image
```
Comment (toast-gear) [inline .github/ISSUE_TEMPLATE/bug_report.yml:14]: I think this needs wording slightly differently, we expect images built on our base to work fine and I can picture some people thinking custom images built from our images (as per the guide) are "customer" runner images which isn't really what we're trying to capture here right?

Comment (mumoshu) [reply .github/ISSUE_TEMPLATE/bug_report.yml:14]: I'm skeptical if everyone can be 100% sure if their "customizations" added on top of the base image won't break anything that ARC expects. Perhaps it's easier for us if we communicate this way. On the other hand, perhaps it's nicer if we could capture folks who make only simple customizations like adding pkgs and commands to the base image.
Do you have any idea about what would be a good sentence to capture only the latter?

Comment (mumoshu) [reply .github/ISSUE_TEMPLATE/bug_report.yml:14]: @toast-gear I missed your edit, sorry! So..

> I'm not using a custom entrypoint in my runner image?

This sounds good!

Can we also add another field to let the user list up all the modifications made to the custom image, which is required when this option is checked?

Comment (mumoshu) [reply .github/ISSUE_TEMPLATE/bug_report.yml:14]:
```suggestion
    - label: I'm not using a custom entrypoint in my runner image
```

**Review by toast-gear — APPROVED** [review]

**Comment by mumoshu** [comment]
Thanks for your review! Merging.

**Pull request merged**